dashboard/app/controllers/lessons_controller.rb

```diff
@@ -5,30 +5,14 @@ class LessonsController < ApplicationController
 
   # GET /lessons/1
   def show
-    @lesson_data = {
-      title: @lesson.localized_title,
-      overview: @lesson.overview,
-      announcements: @lesson.announcements,
-      purpose: @lesson.purpose,
-      preparation: @lesson.preparation
-    }
+    @lesson_data = @lesson.to_camelized_hash
+    @lesson_data[:title] = @lesson.localized_title
```

Comment:
Problem 1: This is kind of a bad sign -- the "title" should really be added inside the serializer.

Reply:
Why isn't it?